src/vesting/library.cairo

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+from starkware.starknet.common.syscalls import get_caller_address
+
+// Project dependencies
+from openzeppelin.security.safemath.library import SafeUint256
+
+// Local dependencies
+from starkvest.library import StarkVest
+
+namespace CarbonableVester {
+    //
+    // Getters
+    //
+
+    func releasable_of{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        account: felt
+    ) -> (amount: Uint256) {
+        // [Check] If no vesting associated to the caller, then return 0
+        let (vesting_count) = StarkVest.vesting_count(account=account);
+        if (vesting_count == 0) {
+            let zero = Uint256(low=0, high=0);
+            return (amount=zero);
+        }
+
+        // [Compute] Sum releasable amount per vesting
+        let vesting_index = vesting_count - 1;
+        let (amount) = _releasable_iter(account=account, vesting_index=vesting_index);
+        return (amount=amount);
+    }
+
+    //
+    // Externals
+    //
+
+    func release_all{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
```

Comment:
I don't really like externals to return something except status since tx responses are not really read by anyone, only the tx success status is commonly read.